packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -47,6 +53,11 @@ export abstract class BaseResource {
 
     const { payload, status, statusText } = fapiResponse;
 
+    if (payload?.meta) {
+      const country = payload?.meta.responseHeaders?.country || 'us';
```

Comment:
Why do we fallback to `us` and check for `: null` right below? 
We probably shouldn't fallback to anything. If we didn't get a country, the value should be `null` and the components should handle `null` accordingly

Reply:
Yes let's allow for nullable country to be set. And let the component handle he fallback. Although we may endup using `us` there in order to display a default selected option for the country extension